paasta_tools/deployd/workers.py

```diff
@@ -58,12 +58,15 @@ def run(self):
             self.log.info("{} processing {}.{}".format(self.name, service_instance.service, service_instance.instance))
             marathon_apps = marathon_tools.get_all_marathon_apps(self.marathon_client, embed_failures=True)
             bounce_timers.setup_marathon.start()
-            return_code, bounce_again_in_seconds = deploy_marathon_service(service=service_instance.service,
-                                                                           instance=service_instance.instance,
-                                                                           client=self.marathon_client,
-                                                                           soa_dir=marathon_tools.DEFAULT_SOA_DIR,
-                                                                           marathon_config=self.marathon_config,
-                                                                           marathon_apps=marathon_apps)
+            try:
+                return_code, bounce_again_in_seconds = deploy_marathon_service(service=service_instance.service,
+                                                                               instance=service_instance.instance,
+                                                                               client=self.marathon_client,
+                                                                               soa_dir=marathon_tools.DEFAULT_SOA_DIR,
+                                                                               marathon_config=self.marathon_config,
+                                                                               marathon_apps=marathon_apps)
+            except Exception:
+                return_code, bounce_again_in_seconds = -1, 60
```

Comment:
I wish there was a simple way to make this exponentially backoff. Is there?
My thinking here is that if we start overloading marathon/mesos and get exceptions, we will continue to hit it harder and harder because these deploys will continue to back up.